src/msg/async/Event.cc

```diff
@@ -140,7 +140,7 @@ void EventCenter::set_owner()
   if (!global_centers) {
     cct->lookup_or_create_singleton_object<EventCenter::AssociatedCenters>(
         global_centers, "AsyncMessenger::EventCenter::global_center");
-    assert(global_centers && !global_centers->centers[idx]);
+    assert(global_centers);
```

Comment:
the problem is after fork we need to spawn threads again, but global_centers->centers[idx] already has value, we can't assert on this